jaxrs-base-server/src/main/java/io/telicent/smart/cache/server/jaxrs/resources/AbstractHealthResource.java

```diff
@@ -33,6 +37,19 @@
 @Path("/")
 public abstract class AbstractHealthResource {
 
+    /**
+     * A cache used to hold the most recently computed Health Status of the application
+     */
+    private static final Cache<String, HealthStatus> CACHED_STATUS =
+            Caffeine.newBuilder().maximumSize(10).initialCapacity(1).expireAfterWrite(Duration.ofSeconds(30)).build();
```

Comment:
```suggestion
            Caffeine.newBuilder().maximumSize(10).initialCapacity(10).expireAfterWrite(Duration.ofSeconds(30)).build();
```

- [ ] Make initial capacity match maximum size
- [ ] Note reasons for cache size being > 1 in Javadoc
- [ ] Make cache parameters constants for clarify